packages/frontend/src/pages/LogIn.js

```diff
@@ -30,6 +30,7 @@ const LogIn = () => {
         //storing token in localStorage
         localStorage.setItem("token", responseData.token);
         localStorage.setItem("userId", responseData.userId);
+        localStorage.setItem("householdId", responseData.householdId);
```

Comment:
Does responseData hold householdId? What is the expected response and what does it hold?

Reply:
You are so right, there is no household id. Unfortunately, even when I add this line it still doesn't work. And yes I made sure all the verbage was houseId (I was curious if having the same name made a difference, apparently not.)
<img width="1142" alt="Screenshot 2024-03-08 at 3 50 24 PM" src="https://github.com/AlexiLynn/RoommateRivalry/assets/102687653/841c8363-5ee0-4bc5-aa6c-17d3bc7b7271">